### PR TITLE
Not logged in

### PR DESCRIPTION
**Pull request by le3ah**
Adds before action to confirm that a user is logged in.  Feature tests for rendering 404 message if a user is not logged in.  All tests passing.  Closes #60 

bouncer

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 class SessionsController < ApplicationController
+  skip_before_action :require_login
+  
   def create
     if user = User.from_google_auth(request.env["omniauth.auth"])
       session[:user_id] = user.id
-      flash[:success] = "Welcome, gardner, we're glad to have you among the roses."
+      flash[:success] = "Welcome, gardener, we're glad to have you among the roses."
```
Comment (abenetka) [inline app/controllers/sessions_controller.rb]: 👏 👏

File: spec/features/visitors/visitor_signs_in_with_google_spec.rb [hunk 2]
```diff
@@ -8,6 +8,6 @@
     expect(page).to have_link("Sign in with Google")
 
     click_link "Sign in with Google"
-    expect(page).to have_content("Welcome, gardner, we're glad to have you among the roses.")
+    expect(page).to have_content("Welcome, gardener, we're glad to have you among the roses.")
```
Comment (abenetka) [inline spec/features/visitors/visitor_signs_in_with_google_spec.rb]: 👍

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -1,8 +1,10 @@
 class SessionsController < ApplicationController
+  skip_before_action :require_login
```
Comment (abenetka) [inline app/controllers/sessions_controller.rb]: This is nice

**Comment by le3ah** [comment]
@Mackenzie-Frey, I had to add logged-in users to files in order for our tests to pass, so that's where the merge conflicts were.  These are resolved now.

**Pull request merged**